web/src/javascript/leaderboardApp.js

```diff
@@ -0,0 +1,16 @@
+import {fetchEntries} from "./leaderboardEntry";
+
+function displayEntries(entries) {
+  const leaderboard = document.getElementById("leaderboard-body");
+  for(const entry of entries) {
+    const row = leaderboard.insertRow(0);
+    row.insertCell(0).innerHTML = entry.team_id;
+    row.insertCell(1).innerHTML = entry.name;
```

Comment:
[`he`](https://github.com/mathiasbynens/he)? [underscore's `escape`](http://underscorejs.org/#escape)? We should at the very least do it on the server-side if we don't do it here.

If we don't want to care about this (since team names are sort of humanly "approved" when creating the teams during the event), I vote that we add a comment `// Bonus point if you XSS this`. If we are to do it wrong, let's do it wrong the right way.

I would still prefer encoding. :-)